ci/integration_tests.sh

```diff
@@ -11,7 +11,7 @@ mkdir -p /tmp/aqueduct_experiments && \
 mkdir -p /tmp/aqueduct_extensions
 
 echo "Building the stack"
-docker-compose -f $PROJECT_ROOT/containers/integration_tests/docker-compose.yml up -d
+docker compose -f $PROJECT_ROOT/containers/integration_tests/docker-compose.yml up -d
```

Comment:
What does this change in practice?

Reply:
Seems like it's a change in GitHub workflows.

> GitHub deprecated v1, and you need to change the command from, e.g., docker-compose build to docker compose build (remove the dash)

I linked the question in the PR description as well.